custom_components/meural/__init__.py

```diff
@@ -23,7 +23,9 @@ async def async_setup(hass: HomeAssistant, config: dict):
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
     """Set up Meural from a config entry."""
     hass.data[DOMAIN][entry.entry_id] = pymeural.PyMeural(
```

Comment:
Don't forget to guide user to set it up again if on old version
```suggestion
    if "email" not in entry.data:
        _LOGGER.warning("Authentication changed. Please set up Meural again")
        return False

    hass.data[DOMAIN][entry.entry_id] = pymeural.PyMeural(
```